data/json/items/armor/suits_protection.json

```diff
@@ -758,7 +758,7 @@
     "repairs_like": "fsurvivor_suit",
     "type": "ARMOR",
     "name": { "str": "hazmat suit" },
-    "description": "An impermeable whole-body garment worn as protection against hazardous materials.  Though very restrictive and fragile, wearing it will provide complete protection against ambient radiation.  It requires a separate gas mask for full protection.",
+    "description": "An impermeable whole-body garment worn as protection against hazardous materials.  Restrictive and fragile, wearing it will provide complete protection against ambient radiation.  It requires a separate gas mask for full protection.",
```

Comment:
Does this actually protect against radiation? (And it shouldn't do full protection in any event vs radiation; vs hazardous chemicals it could - at least briefly.)

Reply:
It has rad-resist, I'm pretty sure? I didn't write the original desc, I just edited it.